PF/Assignment1/VehicleReadMain.java

```diff
@@ -0,0 +1,80 @@
+import java.util.List;
+import java.util.Scanner;
+
+//Main class to perform all operations
+public class VehicleReadMain {
+	public static void main(String args[]) throws VehicleManagementException {
+		Scanner scan = new Scanner(System.in);
+
+		// reading from XML
+		StaxParser objSaxParser = new StaxParser();
+		String filePath = "C:/Users/Anshita/workspace/assignment/Case-Study2/src/vehicle.xml";
+		List<Vehicle> readVehicleFromConfig = objSaxParser.readConfig(filePath);
+		System.out.println("reading value from file");
+		for (Vehicle objVehicle : readVehicleFromConfig) {
+			System.out.println(objVehicle);
+
+			// create table
+			VehicleJdbcHelper.createVehicleTable(objVehicle);
```

Comment:
ACTION: for all vehicleRecords we will execute this method, is it not possible to set a static boolean variable with true in vehicleHelper class to check vehicle is created or not same for child helper class. So we will just check that variable rather than executing query for all records.